README.md

```diff
@@ -1,10 +1,48 @@
-# default-SEM-model
+# model_seg_rat_axon-myelin_sem
+---
+
+## Model overview
+![image of segmentation obtained from this model](sem_model_preview.png)
```

Comment:
The images are in a different orientation than the original PNG, is that expected?

Reply:
Yes but only for cosmetic reasons because I wanted the figure to fit more or less in a square. If you think that's confusing I can change it.

Reply:
No it's allright, I was just wondering